api/proxy/config/v2/eigendaflags/cli.go

```diff
@@ -29,6 +29,7 @@ var (
 	)
 	ServiceManagerAddrFlagName        = withFlagPrefix("service-manager-addr")
 	BLSOperatorStateRetrieverFlagName = withFlagPrefix("bls-operator-state-retriever-addr")
```

Comment:
think we should mark these as deprecated and hint people to start moving to the eigenda-directory? Prob want to do it across all of our services (apiserver, nodes, etc). Everything in this PR. And eventually we'll remove all other address flags and only keep the eigenda-directory.

not sure if urfave has a way to mark flags as deprecated. If not then we might need to manually log a warn. cc @dmanc who will be updating our helm charts to use these new flags. Can you also add this as a note to this PR description? This kind of external config changes are very important to highlight.

Reply:
i made a linear [ticket](https://linear.app/eigenlabs/issue/EGDA-1647/proxy-eigenda-config-flag-deprecate-legacy-addresses) for proxy, and for all services [ticket](https://linear.app/eigenlabs/issue/EGDA-1644/remove-all-address-configs-and-only-keep-eigendadirectory). I will add notes to the PR description